public/index.html

```diff
@@ -101,6 +132,37 @@ <h2>PayJoin here to open these channels</h2>
 				channels.removeChild(channels.children[channels.children.length - 2]);
 			}
 
+			async function get_recommended_channels(){
+				document.getElementsByClassName('recommendation-button-wrapper')[0].classList.add('invisible')
```

Comment:
Could make this change button text to `Refresh Recommended Channels` and give random ones from the api. Or maybe leave for another PR

Reply:
Lets save this for a follow up PR. https://github.com/chaincase-app/nolooking/issues/66